notebooks/xvcfsubsample/main.py

```diff
@@ -0,0 +1,89 @@
+# publish to: "terra-notebook-utils-tests" "test"
+import os
+import herzog
+from firecloud import fiss
+
+with herzog.Cell("markdown"):
+    """
+    # VCF Subsample Workflow Input Preparation
+
+    This notebook demonstrates the creation of Terra data tables as input to xvcfsubsample workflows, referencing input
+    and output VCFs with DRS URIs or Google Storage URLs. It should be used with xvcfsubsample version
+    [0.0.3](https://dockstore.org/workflows/github.com/DataBiosphere/xvcfsubsample/xvcfsubsample:v0.0.3?tab=info)
+
+    ### Workflows
+
+    Import the [Dockstore](https://dockstore.org) workflows into your workspace using the "NHLBI Biodata Catalyst" launch button:
+     - [xvcfsubsample](https://dockstore.org/workflows/github.com/DataBiosphere/xvcfsubsample/xvcfsubsample:v0.0.3?tab=info)
+
+    ### Input File Requirements
+
+    - Input files _must_ be referenced with either DRS URIs or Google bucket URLs.
+    - Output files _must_ be Google bucket URLs. The target bucket will usually be your workspace bucket, but it can be
+      any bucket for which you have write access.
+    """
+
+os.environ['WORKSPACE_NAME'] = "terra-notebook-utils-tests"
+os.environ['WORKSPACE_BUCKET'] = "gs://fc-9169fcd1-92ce-4d60-9d2d-d19fd326ff10"
+os.environ['GOOGLE_PROJECT'] = "firecloud-cgl"
+
+with herzog.Cell("python"):
+    #%pip install --upgrade --no-cache-dir terra-notebook-utils
+    pass
+
+with herzog.Cell("python"):
+    import os
+    from terra_notebook_utils import table
+
+    workspace = os.environ['WORKSPACE_NAME']
+    workspace_namespace = os.environ['GOOGLE_PROJECT']
+    workspace_bucket = os.environ['WORKSPACE_BUCKET']
+
+with herzog.Cell("markdown"):
+    """
+    ## Option A: VCF subsample workflow input for DRS URIs
+    This is a typical workflow preparation for merging TOPMed VCFs _without_ downloading them to your workspace.
```

Comment:
replace merging with subsampling on this line.

Reply:
done